Rakefile

```diff
@@ -17,3 +17,10 @@ namespace :test do
     t.pattern = "test/mysql_test.rb"
   end
 end
+
+namespace :noncompulsory_tests do
```

Comment:
I like the consistency of `test:redshift` with `test:postgresql` and `test:mysql`. Maybe we could keep `test:redshift` and modify `test` to use the `test_files` option of Rake::TestTask. http://ruby-doc.org/stdlib-2.0.0/libdoc/rake/rdoc/Rake/TestTask.html